src/rotator_library/score_engine.py

```
@@ -86,142 +140,68 @@ def is_provider_available(
+            return True
+        try:
+            health = self.telemetry.get_provider_health(provider, model)
+            if not health.get("is_healthy", True):
+                return False
+            if health.get("consecutive_failures", 0) > 5:
+                return False
+            is_limited, _ = self.telemetry.check_rate_limit(provider, model or "*")
+            if is_limited:
+                return False
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)